lib/ip.go

```diff
@@ -193,25 +201,27 @@ IP:
 // RandIP6Range returns a list of randomly generated IPv6 addresses within
 // the range specified by `startIP` and `endIP`.
```

Comment:
We still need to update this part of the comment - please read it carefully to see why.

The same applies for the v4 function.